cli/script/command-executor.ts

```diff
@@ -1347,6 +1365,47 @@ function serializeConnectionInfo(accessKey: string, preserveAccessKeyOnLogout: b
     log(`\r\nSuccessfully logged-in. Your session file was written to ${chalk.cyan(configFilePath)}. You can run the ${chalk.cyan("code-push logout")} command at any time to delete this file and terminate your session.\r\n`);
 }
 
+function sessionList(command: cli.ISessionListCommand): Promise<void> {
+    throwForInvalidOutputFormat(command.format);
+
+    return sdk.getAccessKeys()
```

Comment:
Since we're now going with this thought that access keys and session keys should just appear completely different to the end users, I'll go ahead and just create separate TypeScript interfaces for each of them.